components/03-modules/active-filters/active-filters.hbs

```diff
@@ -0,0 +1,21 @@
+<ul class="active-filters list">
+    {{# each filters }}
+        <li class="active-filters__item list__item">
+            <button type="button"
+                    class="active-filters__remove"
+                    aria-label="{{ariaLabel}}"
+            >
+                <svg class="active-filters__remove-icon">
+                    <use xlink:href="{{ svg iconId }}"></use>
+                </svg>
+            </button>
+            <span class="active-filters__label">{{label}}&nbsp;:&nbsp;</span>
```

Comment:
1) Why we are using here `&nbsp;` ?
2) When checking what it could be used fore I edited the value and get this effect
[![Screenshot from Gyazo](https://gyazo.com/00d207416799243ec2085d55a0759452/raw)](https://gyazo.com/00d207416799243ec2085d55a0759452) - should it be like that?

Reply:
1. weird question, I guess you know what is non-breaking spaces are used for.
2. Fixed, now both label and value won't affect icon size, and won't overflow the container either: 
![image](https://user-images.githubusercontent.com/12099511/33607267-d217d44a-d9c0-11e7-9dcf-099cc2422d59.png)

Reply:
1. I didn't notice colon so topic closed.
2. shouldn't be "X" aligned to top ? if some labels would be like in second example?